test/system/280-update.bats

```diff
@@ -127,4 +127,27 @@ device-write-iops   = /dev/zero:4000 | - | -
     fi
 }
 
+@test "podman update - set restart policy" {
+    TMPD=$(mktemp -d podman-update-test.restart.XXXXXXXX)
+    touch ${TMPD}/sentinel
+    run_podman run --name testctr -v ${TMPD}:/testdir -d alpine sh -c "touch /testdir/alive; while test -e /testdir/sentinel; do sleep 0.1; done;"
```

Comment:
```suggestion
    run_podman run --name testctr -v ${TMPD}:/testdir -d $IMAGE sh -c "touch /testdir/alive; while test -e /testdir/sentinel; do sleep 0.1; done;"
```
just noticed this now but this should be the normal testimage, I don't think it is causing the issue with the hang but it should prevent a unnecessary pull

Reply:
Agree. Also please use `$PODMAN_TMPDIR`  (you can assign a local variable to make it shorter). Your `mktemp` writes in cwd, which is a bad idea. Especially since we already have a tmpdir.

Reply:
${PODMAN_TMPDIR} is the actual fix here, the current `mktemp -d podman-update-test.restart.XXXXXXXX` creates a relative directory (without slash) so podman run -v assumed this to be named volume instead.